theme/static/style/style.css

```diff
@@ -177,6 +177,12 @@ nav button {
     margin-bottom: 2rem;
 }
 
+/* make blog posts' hero images responsive */
+.article p:first-child img {
```

Comment:
Remove `p:first-child` b/c we want this to apply to all images in the page. For example on
http://kiwitcms.org/blog/atodorov/2020/01/17/project-roadmap-2020/ the image follows in the 2nd paragraph.

On http://kiwitcms.org/blog/atodorov/2019/11/11/how-to-override-bug-details-in-kiwi-tcms/ there are multiple images in the text which also need to have max-width applied to them.

Reply:
I'm just thinking, since we're applying responsiveness to all images on the
page, can I move this rule into the rule-set for images, which has already been
defined on line 90
https://github.com/mfonism/kiwitcms.github.io/blob/2263b936c9e1417474d4fa232a6bd88030d45959/theme/static/style/style.css#L90-L93

This will minimize the changes.

Reply:
Probably yes. You should check how the home page looks like after you do this. Also search the contents for the "<img" HTML tag - some pages use HTML instead of Markdown to include he images and they also change style especially where there is text floating around the image. If they look OK then the changes you propose don't break anything.